src/bun.js/bindings/Path.cpp

```diff
@@ -184,27 +185,30 @@ static JSC::JSObject* createPath(JSGlobalObject* globalThis, bool isWindows)
             Path_functionToNamespacedPath, ImplementationVisibility::Public),
         0);
 
-    if (isWindows) {
-        path->putDirect(vm, clientData->builtinNames().sepPublicName(),
-            JSC::jsString(vm, WTF::String("\\"_s)), 0);
-        path->putDirect(vm, clientData->builtinNames().delimiterPublicName(),
-            JSC::jsString(vm, WTF::String(";"_s)), 0);
-    } else {
-        path->putDirect(vm, clientData->builtinNames().sepPublicName(),
-            JSC::jsString(vm, WTF::String("/"_s)), 0);
-        path->putDirect(vm, clientData->builtinNames().delimiterPublicName(),
-            JSC::jsString(vm, WTF::String(":"_s)), 0);
-    }
-
     return path;
 }
 
 } // namespace Zig
 
-extern JSC__JSValue Bun__Path__create(JSC::JSGlobalObject* globalObject, bool isWindows)
+namespace Bun {
+    
+JSC::JSValue createNodePathBinding(Zig::GlobalObject* globalObject)
 {
     JSC::VM& vm = globalObject->vm();
+    auto binding = constructEmptyArray(globalObject, nullptr, 2);
```

Comment:
Would this be better:
```c++
 binding = JSC::JSArray::tryCreateUninitializedRestricted(
            object,
            globalObject->arrayStructureForIndexingTypeDuringAllocation(JSC::ArrayWithContiguous),
            2);
            ```

Reply:
<img width="1538" alt="image" src="https://github.com/oven-sh/bun/assets/24465214/883d71af-6375-4066-8d3b-35513af27240">

reading this comment, tbh no.

Reply:
eeee. that's some disclaimer. We use `JSC::JSArray::tryCreateUninitializedRestricted(` in places, I wonder if we should rethink it.